### DISCUSSION TITLE
Include emoticons feedback in the feedback form

### DISCUSSION THREAD

**Discussion by ayushbits** [BetaHuhn/feedback-js#79] [Ideas]
In addition to email and description, can we add emoticons in the forms as well ?

**Comment by BetaHuhn** [comment]
Do you mean adding emoticons as a rating?

**Reply by ayushbits** [reply]
Thanks for the quick reply @BetaHuhn . 
Yes, I mean a rating kind of thing in the form itself and rating can be in the form of emoticons.
![image](https://user-images.githubusercontent.com/30416518/148505271-c6d7fc3f-a9e4-4bb3-91b5-6dceddfa9fb0.png)

**Reply by BetaHuhn** [reply]
This is not something I am currently planning on adding, sorry. The form is meant to be very simple, both for a user and for the developer. Ratings are always a bit more complicated to customise and handle.
